File: src/main/resources/static/scripts/service/media.js
/**
 * Balero CMS Project: Proyecto 100% Mexicano de código libre.
 * Página Oficial: http://www.balerocms.com
 *
 * @author      Anibal Gomez <anibalgomez@icloud.com>
 * @copyright   Copyright (C) 2015 Neblina Software. Derechos reservados.
 * @license     Licencia BSD; vea LICENSE.txt
 */

angular.module('MediaService', ['ngResource'])
    .factory('MediaService', function($resource) {
        return {
            getUploads: function() {
                return $resource('../admin/api/uploads', {}, {
                    query: { method: 'GET', params: {}, isArray: true}
                })
            }
        };
    });

File: src/main/java/com/neblina/balero/domain/Media.java
/**
 * Balero CMS Project: Proyecto 100% Mexicano de código libre.
 * Página Oficial: http://www.balerocms.com
 *
 * @author Anibal Gomez <anibalgomez@icloud.com>
 * @copyright Copyright (C) 2015 (20/10/15) ) Neblina Software. Derechos reservados.
 * @license Licencia BSD; vea LICENSE.txt
 */

package com.neblina.balero.domain;

public class Media {

    private String fileName;
    private int width;
    private int height;

    public String getFileName() {
        return fileName;
    }

    public void setFileName(String fileName) {
        this.fileName = fileName;
    }

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    @Override
    public String toString() {
        return "Media [" +
                "fileName=" + this.fileName + "," +
                "width=" + this.width + "," +
                "height=" + this.height
                + "]";
    }

}
